src/esphomelib/sensor/tx20_component.h

```diff
@@ -0,0 +1,59 @@
+
+#include "esphomelib/defines.h"
+
+#ifdef USE_TX20
+
+#include "esphomelib/sensor/sensor.h"
+#include "esphomelib/text_sensor/text_sensor.h"
+
+ESPHOMELIB_NAMESPACE_BEGIN
+
+namespace sensor {
+
+extern const char UNIT_KMH[];
+extern const char ICON_WIND_SPEED[];
+extern const char ICON_WIND_DIRECTION[];
+extern const char EMPTY[];
+
+using TX20WindSpeedSensor =
+    sensor::EmptyPollingParentSensor<1, ICON_WIND_SPEED, UNIT_KMH>;
+using TX20WindDirectionSensor =
+    sensor::EmptyPollingParentSensor<1, ICON_WIND_DIRECTION, EMPTY>;
+using TX20WindDirectionTextSensor = text_sensor::TextSensor;
+
+/// This class implements support for the TX20 WindSpeed+WindDirection sensor.
+class TX20Component : public PollingComponent {
```

Comment:
Why make this a `PollingComponent` when the sensor is asynchronous (using interrupts) ?